router/core/errors.go

```diff
@@ -316,3 +316,14 @@ func writeOperationError(r *http.Request, w http.ResponseWriter, requestLogger *
 		writeRequestErrors(r, w, http.StatusInternalServerError, graphqlerrors.RequestErrorsFromError(errInternalServer), requestLogger)
 	}
 }
+
+type ExprWrapError struct {
```

Comment:
This is to circumvent this problem described here:
https://github.com/wundergraph/cosmo/pull/1545#discussion_r1936100140

While testing further I noticed that there can be other error types, so it was safer to wrap the error in a type we can control, so we have a deterministic error type outputted from the `exprlang` expression.

(I didn't come across anything atm, but if there is a better existing type to use, we can reuse that)